### ISSUE TITLE
Alt text on user images should be required

### ISSUE THREAD

**Issue by thejimbirch** [badcamp/badcamp-2018#131]
For WCAG compliance, all images should have alt text.  My guess is that there is an image field on the user profile where it can be enabled and required.

**Comment by JQTNguyen** [comment]
This has been performed on LIVE, and will be exported with the next Pull Request.
